### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by moses369**
Found a Start Rating Library that works, edited your filed to include it. Got rid of the old package.

### CODE REVIEW

File: components/Carosel/carousel2.js [hunk 1]
```diff
@@ -40,8 +40,7 @@ const T_shirt_line = () => {
         );
       }
         const tShirtArray = tshirts.data.tshirt;
-        console.log(tShirtArray);
-        const starRating = starPower();
+        // console.log(tShirtArray);
```
Comment (moses369) [inline components/Carosel/carousel2.js:43]: - Got rid of old star rating import, and added the one I made that works.
- Commented out your log to not blow up the console

File: components/Carosel/carousel2.js [hunk 2]
```diff
@@ -60,7 +59,7 @@ const T_shirt_line = () => {
                       <img className={styles.tShirtImage} src={tShirtArray[index].img} alt="sample" />
                       <div>
                         <h5 className={styles.tShirt_description}>{tShirtArray[index].description}</h5>
-                        <h6 className={styles.starSpan}>{starRating}</h6>
+                        <h6 className={styles.starSpan}>{<StarRating/>}</h6>
```
Comment (moses369) [inline components/Carosel/carousel2.js:62]: Replaced your starRating import with the one I created

File: components/Carosel/starRating.js [hunk 3]
```diff
@@ -1,28 +0,0 @@
...
-      allowHover={false}
-      allowHalfIcon
-      size={20}
-      
-     />
-    </div>
-  )
-}
-
-export default starPower;
```
Comment (moses369) [inline components/Carosel/starRating.js:28]: Created new star rating component, deleted this one as it is no longer necessary

File: components/StarRating.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      <Ratings.Widget />
+      </Ratings>
+  )
+}
+StarRating.defaultProps = {
+    rating:0,
+    size:'1.2rem',
+    spacing:'.2rem',
+}
+export default StarRating
```
Comment (moses369) [inline components/StarRating.js:27]: New star rating component, takes in three props, 
- rating: number (value of the rating) 
- size: string (how large the stars will be)
- spacing: string (how far apart each star will be)

**Review by CountDown2Extinction — APPROVED** [review]

**Pull request merged**